images/nginx/rootfs/build.sh

```diff
@@ -92,8 +92,8 @@ export LUA_RESTY_CACHE=0.13
 # Check for recent changes: https://github.com/openresty/lua-resty-core/compare/v0.1.27...master
 export LUA_RESTY_CORE=0.1.27
 
-# Check for recent changes: https://github.com/utix/lua-resty-cookie/compare/9533f47...master
-export LUA_RESTY_COOKIE_VERSION=9533f479371663107b515590fc9daf00d61ebf11
+# Check for recent changes: https://github.com/utix/lua-resty-cookie/compare/12136f5...master
+export LUA_RESTY_COOKIE_VERSION=12136f546ec4222e198c2d7c73dd83fc3539df32
```

Comment:
can you please update nginx-1.25 folder as well please?

Reply:
Done, i'll note that it was still using the cloudflare build so this change bumps it several versions up.